### PR TITLE
docs: add WHERE expression syntax to operators.md

### PR DESCRIPTION
**Pull request by spena**
### Description 

Add a WHERE expression syntax to the operators.md document.

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/developer-guide/ksqldb-reference/expressions.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+title: ksqlDB Expressions
+tagline:  ksqlDB expressions for queries
+description: Expressions to use in  ksqlDB statements and queries
+keywords: ksqlDB, expression
+----------------------------
+
+Expressions
+===========
+
+ksqlDB supports the following expression syntax in value expressions.
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-reference/expressions.md:12]:
```suggestion
ksqlDB supports the following syntax in value expressions.
```

File: docs/developer-guide/ksqldb-reference/expressions.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: ksqlDB Expressions
+tagline:  ksqlDB expressions for queries
+description: Expressions to use in  ksqlDB statements and queries
+keywords: ksqlDB, expression
+----------------------------
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-reference/expressions.md:7]:
```suggestion
---
```

File: docs/developer-guide/ksqldb-reference/operators.md [hunk 3]
```diff
@@ -2,9 +2,52 @@
 layout: page
 title: ksqlDB Operators
 tagline:  ksqlDB operators for queries
-description: Operators to use in  ksqlDB statements and queries
+description: Operators to use in ksqlDB statements and queries
 keywords: ksqlDB, operator
----
+--------------------------------------
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-reference/operators.md:7]:
```suggestion
---

```

File: docs/developer-guide/ksqldb-reference/expressions.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
+---
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-reference/expressions.md:1]: This is great! Can you add a link in mkdocs.yml?

Comment (spena) [reply docs/developer-guide/ksqldb-reference/expressions.md]: I deleted the expressions.md. I initially was going to use it, but I ended up putting the syntax in the operators.md. I just forgot to delete the file.

**Comment by spena** [comment]
Thanks @JimGalasyn . I made the changes.

**Comment by spena** [comment]
@JimGalasyn Can I merge this PR?

**Comment by JimGalasyn** [comment]
@spena go for it!

**Pull request merged**